src/StickyParallaxHeader.tsx

```diff
@@ -66,44 +149,49 @@ class StickyParallaxHeader extends Component {
             ? this.tabsScrollPosition[currentPage]
             : scrollHeight;
         const scrollNode = getSafelyScrollNode(this.scroll);
-        scrollNode.scrollTo({ y: scrollTargetPosition, duration: 1000 });
+        scrollNode?.scrollTo({ y: scrollTargetPosition });
       }, 250);
     }
   }
 
   componentWillUnmount() {
+    // https://reactnative.dev/docs/animatedvaluexy#removealllisteners
```

Comment:
Is this comment necessary?

Reply:
It explains why we have !ts-ignore, I will add additional comment for that